lametro/management/commands/generate_tag_analytics.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from tqdm import tqdm
+from django.core.management.base import BaseCommand
+
+from lametro.models import LAMetroBill
+
+
+class Command(BaseCommand):
+
+    help = "This command produces a CSV file that lists each Board Report's tags."
+
+    def handle(self, *args, **options):
+
+        with open('tag_analytics.csv', 'w', newline='') as f:
+            writer = csv.writer(f, delimiter=',')
+            files = []
+            for bill in LAMetroBill.all_objects.all():
```

Comment:
I see what you're doing here, but we only want this to return the board reports on the site, i.e., those returned by the LAMetroBillManager.